docs/tutorials/podman_pasta_ipv6_with_snid_ipv4.md

```diff
@@ -0,0 +1,761 @@
+                key-directory "/etc/bind/keys";          # this sets the directory in which this zone's DNSSEC keys will be stored
+                inline-signing yes;                      # this allows BIND to transparently update our signed zone file
+                                                         # whenever we change the unsigned file
+        };
+
+
+        zone "." {
+                type forward;
+                forward only;
+                forwarders {
+			                  // Use Google Public Recursive Name Servers
+                        8.8.8.8;
+			                  8.8.4.4;
+			                  2001:4860:4860::8888;
+			                  2001:4860:4860::8844;
+                };
+        };
+
+
+        // Default Zones
+        include "/etc/bind/named.conf.default-zones";
+};
+```
+
+Then it's pretty much a standard BIND Zone Configuration File:
+```
+; MYDOMAIN.TLD.
+
+$ORIGIN MYDOMAIN.TLD.
+$TTL    5m
+@       IN      SOA	ns1.MYDOMAIN.TLD. ns2.MYDOMAIN.TLD. (
+	                      YYYYMMDD01					; Serial (10 digits)
+                        604800							; Refresh
+                        3600							  ; Retry
+			                  ;86400							; Retry
+                        2419200							; Expire
+                        604800)							; Negative Cache TTL
+;
+
+; Authoritative nameservers
+			                  IN      NS      ns1
+			                  IN	    NS	    ns2
+
+; Defaults 
+                        IN      A       198.51.100.10
+
+; Nameservers
+ns1                     IN      A       198.51.100.10
+ns1			                IN	    AAAA	  2001:db8:0000:0001:0000:0000:0000:0003
+
+; For now NS2 is just the same as NS1
+ns2                     IN      A       198.51.100.10
+ns2			                IN	    AAAA	  2001:db8:0000:0001:0000:0000:0000:0003
+
+; ##########################################################################################
+
+podmanhost01		        IN	    A	      198.51.100.10
+podmanhost01        	  IN	    AAAA	  2001:db8:0000:0001:0000:0000:0000:0100
+
+whoami			            IN	    A       198.51.100.10
+whoami			            IN      AAAA    2001:db8:0000:0001:0000:0000:0001:0001
+
+```
+
+Add to `/etc/systemd/resolved.conf` on the DNS Server Host:
+```
+[Resolve]
+# Use local BIND Name Server, which will forward to Hetzner Recursive Name Servers for external (non-managed) Domains
+DNS=127.0.0.1
+Domains=MYDOMAIN.TLD
+DNSSEC=yes
+DNSOverTLS=opportunistic
+
+# Set to NO since there is already a DNS Server on the System (BIND/NAMED)
+DNSStubListener=no
+```
+
+Add to `/etc/systemd/resolved.conf` on the Podman Host:
+```
+[Resolve]
+DNS=172.16.1.3 2001:db8:0000:0001:0000:0000:0000:0003
+#FallbackDNS=
+Domains=MYDOMAIN.TLD
+DNSSEC=yes
+DNSOverTLS=opportunistic
+```
+
+On both Hosts, run:
+```
+systemctl restart systemd-resolved
+```
+
+In this way, the correct IPv4 <-> IPv6 Translation should be performed and the correct Application should now be able to be reacheable.
```

Comment:
Too specific and needs to go.

Reply:
Moved to https://github.com/luckylinux/podman-notes/blob/main/dns/dns_setup.md. Kept just the Section "DNS Setup" and Link to external Notes.